src/poetry_plugin_export/command.py

```diff
@@ -82,9 +82,22 @@ def handle(self) -> None:
                 "</warning>"
             )
 
+        # Checking extras
+        extras = []
+        for extra in self.option("extras"):
+            if " " in extra:
+                extras += [e.strip() for e in extra.split(" ")]
+            else:
+                extras.append(extra)
+        invalid_extras = set(
```

Comment:
suggest
```python
invalid_extras = set(extras) - self.poetry.package.extras.keys()
```
is a bit cleaner?

Actually `extras` should probably be `set[str]` rather than `list[str]` all along and throughout this plugin, though it's not reasonable to ask this MR to be responsible for that.